debian/jellyfin-ffmpeg.postinst

```diff
@@ -3,3 +3,9 @@
 # Symlink our old locations to the new locations
 # to aid in transitioning around the 10.3.0 release
 ln -s /usr/lib/jellyfin-ffmpeg /usr/share/jellyfin-ffmpeg
+
+# Symlink AMD VAAPI drivers to the dri folder
+if [ $(uname -m) = "x86_64" ]; then
+	ln -s /usr/lib/x86_64-linux-gnu/dri/r600_drv_video.so /usr/lib/jellyfin-ffmpeg/dri
+	ln -s /usr/lib/x86_64-linux-gnu/dri/radeonsi_drv_video.so /usr/lib/jellyfin-ffmpeg/dri
+fi
```

Comment:
When ffmpeg uses vaapi, it will look for an environment variable `LIBVA_DRIVERS_PATH` (usually located in `/usr/lib/dri`). Since the location of the Intel VAAPI driver in the deb package is `jellyfin-ffmpeg/dri`, I modified this part of `va.c` to make `libva` skip the search for the driver that comes with the system and use the driver included in the deb. Therefore, we also need to symlink the AMD VAAPI drivers here.

If these environment variables are still used, once the user destroys them or makes changes during system update installation, `libva` in the deb package will not find a custom driver path.

Reply:
I suggest then replacing `LIBVA_DRIVERS_PATH` in the compiled `libva` with `/usr/lib/jellyfin-ffmpeg/dri:/usr/lib/x86_64-linux-gnu/dri/:/usr/lib/dri`, this would make this hack not needed

Reply:
Does this mean that libva will look for `.so` in a colon-separated order?

Reply:
No idea :) But that's how typical `FOO_PATH` stuff works on Linux.
Let me quickly read their source.

Reply:
Yes, it works just as I expected, see here and below: https://github.com/intel/libva/blob/master/va/va.c#L442-L458